osm2streets/src/types.rs

```diff
@@ -120,52 +120,36 @@ pub enum ConflictType {
     Cross,
 }
 
-/// What kind of feature an Intersection actually represents. Any connection between roads in the
-/// network graph is represented by an Intersection, but many of them are not "intersections" in
-/// the traditional sense.
-///
-/// This type might be better named IntersectionType and the distinction between MultiConnection,
-/// Merge and Crossing is of dubious value and will probably change in the future.
+/// What kind of feature an `Intersection` actually represents. Any connection between roads in the
```

Comment:
Other optional style thing: some of these types could live in `intersection.rs`. I don't think we need loads of organization so far, but `road.rs` and `intersection.rs` have a reasonably clear split for many things

Reply:
I will look at moving some bits around next.